lib/rubocop/cop/rails/enum_syntax.rb

```diff
@@ -107,6 +96,10 @@ def enum_name(elem)
           end
         end
 
+        def option_key?(pair)
+          pair.key.source[0] == '_'
```

Comment:
Hmm, wouldn't this approach result in false negatives for cases like `enum _key, {val1: 0, val2: 1}`?
It might be more appropriate to exclude only the key names that are meaningful as options, such as `_prefix`, `_suffix`, `_scopes`, and `_default`. What do you think?

Reply:
I guess that's right, yeah.

I wonder if anyone would actually have a db schema where this would matter but it's easy to support and rails doesn't have a problem with it.